src/sas/qtgui/Plotting/Slicers/SectorSlicer.py

```diff
@@ -273,6 +274,55 @@ def draw(self):
         """
         self.base.draw()
 
+    def getSlice(self, nbins=None):
```

Comment:
This overlaps the `_post_data` method on line 121 quite a lot. Did you just want to expose the private method in some meaningful way?

Reply:
Yes that's exactly the function I wanted but I needed it to be public and output the data. Also I was afraid if I made any changes to it id end up breaking it. Ill try this again by creating a new function that calls this and then returns the data.

Reply:
If this is to capture the slice data and use it elsewhere, I'd suggest you couple my above suggestion with:

```
def captureSlice(self, nbins=None):
    new_plot = self._post_data(nbins, show_plots=False)
    # Do anything else you need to do with the plot that isn't done in _post_data
    return new_plot
```

Reply:
Rewritten as proposed